gradle.propertiess

```diff
@@ -0,0 +1,11 @@
+org.gradle.caching=true
```

Comment:
Uhhm I believe you mistyped the name of this file

Reply:
thanks, good catch

Reply:
Feel free to merge afterwards